core/os/android/adb/commands.go

```diff
@@ -496,29 +496,49 @@ func (b *binding) DriverVersionCode(ctx context.Context) (int, error) {
 	return ip.VersionCode, err
 }
 
-// HasGpuProfilingSupportInSystemImage returns whether system image has GPU profiling support.
-func (b *binding) HasGpuProfilingSupportInSystemImage(ctx context.Context) (bool, error) {
-	supported, err := b.SystemProperty(ctx, systemImageGpuProfilerSupportProperty)
+// PrepareGpuProfiling queries GPU profiling support, and when profiling is supported it sets up
+// the device for profiling of installedPackage app. It returns:
+// - a bool which is true when GPU profiling is supported and the setup is done without errors
+// - a string that contains the name of the package where to find profiling layers
```

Comment:
let's explicitly call out that empty string means BSP driver by default if GPU profiling is supported.

Reply:
done